src/put.js

```diff
@@ -59,6 +61,34 @@ module.exports = {
           res.status(500).send(err.message)
         })
     })
+
+    putMetaHandler = (context, path, value, cb) => {
+      let defaults
+
+      try {
+        defaults = skConfig.readDefaultsFile(app)
+      } catch (e) {
+        defaults = {
+          vessels: {
+            self: {}
+          }
+        }
+      }
+      const full = context + '.' + path
```

Comment:
`full` refers to _the full model_ in my head, maybe `pathWithContext`?

Reply:
done